packages/story-editor/src/karma/fixture/containers/canvas.js

```diff
@@ -72,7 +72,7 @@ export class Canvas extends Container {
 
   get quickActionMenu() {
     return this._get(
-      this.getByRole('menu'),
+      this.getByTestId('Element quick actions'),
```

Comment:
Not a big fan of this one - we should avoid using test-ids completely (as they're a "fake" way to recognize elements).

I do see that the quick-action menu element applies the label to the wrong element. So if we move [this line](https://github.com/GoogleForCreators/web-stories-wp/blob/7722b81051310337bfd0cb186f14a21a143438e6/packages/design-system/src/components/contextMenu/contextMenu.js#L61) to the `Menu` a few lines below (or maybe just duplicate it to avoid breaking something else), the following should work:

```js
      this.getByRole('menu', { name: /Group of available options/ }),
```